### DISCUSSION TITLE
Move to private, resilient code hosting

### DISCUSSION THREAD

**Discussion by Bugsbane** [markqvist/Reticulum#95] [Ideas]
The project bills itself as private and resilient, yet hosts and issuetracks on GitHub, which is neither.

Suggestion: move to privately hosted GIT repo or at least Codeburg.

**Comment by tgrosinger** [comment]
I'm a big fan of https://sourcehut.org and generally use it for anything that doesn't require being in Github.

**Comment by markqvist** [comment]
I definitely agree that there are platforms that could be much better than GitHub privacy-wise, and moving to one of those is an interesting prospect. I am not going to prioritise investigating it right now though, but will leave this issue here until I have the time to do so.

Thanks for the suggestion!

**Comment by markqvist** [comment]
I am moving this to a discussion, as it is not really an issue with the program per se.

**Comment by marek22k** [comment]
In addition, this would make it possible to build Reticulum on IPv6-only nodes. IPv6-only nodes are sometimes cheaper and sufficient - it would be nice if you could clone the Git repo there directly.

+1 for codeberg.org

**Comment by LinuxinaBit** [comment]
+1 for codeberg

**Comment by markqvist** [comment]
Thanks a lot for all the suggestions. For quite some time now, the main repositories for all Reticulum-related projects have actually been hosted on a private, self-hosted Gitea instance, and the GitHub repos are public mirrors of those. Anyone who has already contributed or wants to participate in Reticulum development are welcome to use this instance instead of GitHub, if the prefer to. Just contact me over LXMF or Matrix, and I can give you the details.
